LMT/lmtanalysis/Measure.py

```diff
@@ -58,6 +58,12 @@
 ''' Cage center in 50x50cm area'''
 cageCenterCoordinates50x50Area = Point( 256, 208 )
 
+''' Size of arena in cm '''
+ARENA_SIZE = 50
+
+''' Margin to define center region in cm (chosen to have same area as non-center)'''
+CENTER_MARGIN = 7.32
```

Comment:
Is that the area between the floor and the top of the wall ?

Reply:
that's only a margin to define an inner central area of the arena.
so the 50x50 has an inner square of size (50-2*7.32)^2